deepchem/models/tensorgraph/symmetry_functions.py

```diff
@@ -413,20 +413,21 @@ def __init__(self,
 
   def create_tensor(self, in_layers=None, set_tensors=True, **kwargs):
     """ Generate Radial Symmetry Function """
+    init_fn = initializations.get(self.init)  # Set weight initialization
```

Comment:
The Layer was unable to pickle the initialization function.  Instead I pickle the initialization string, and convert it to a function when building the graph.